src/Query/ActiveQuery.php

```diff
@@ -105,7 +105,7 @@ public function sqlStatement(QueryParameters $parameters = null): string
 
         return $this->driver->getQueryCompiler()->compile(
             $parameters ?? new QueryParameters(),
-            $this->prefix,
+            $this->prefix ?? "",
```

Comment:
I suggest using single quotes instead of double quotes. It’ll be more consistent